airbyte-webapp/src/hooks/useLoadingState.tsx

```diff
@@ -37,11 +36,9 @@ const useLoadingState = (): {
           feedbackAction();
         }
       }, 2000);
-    } catch (error) {
-      const message = error?.message || formatMessage({ id: "notifications.error.noMessage" });
```

Comment:
Why remove?

Reply:
Previously we had a title and message, we were showing that smth went wrong and message with details(http error message) and now we need to choose what to show.
<img width="644" alt="image" src="https://user-images.githubusercontent.com/20929344/205398223-59ffd47f-d1ab-4ba1-beb4-904d873de469.png">

I guess it can confuse the user if we only show a message like in the screenshot above.
Or is it okay to show a message like this one?

Reply:
I think this is probably fine.